types/filters/parse.go

```diff
@@ -8,6 +8,8 @@ import (
 	"fmt"
 	"regexp"
 	"strings"
+
+	versionpkg "github.com/docker/docker/pkg/version"
```

Comment:
as I mentioned, we cannot import from `docker/docker`. It's a restriction of this package.